src/Cohesity.Powershell/Scripts/View/Set-CohesityViewWhitelist.ps1

```diff
@@ -0,0 +1,61 @@
+function Set-CohesityViewWhitelist {
+    <#
+        .SYNOPSIS
+        Add whitelist IP(s) for a given view.
+        .DESCRIPTION
+        Add whitelist IP(s) for a given view.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Set-CohesityViewWhitelist -ViewName view1 -ViewWhitelist $viewWhitelist
+        Get the whitelist as follows, $viewWhitelist = Get-CohesityViewWhitelist -ViewName view1
+        Set whitelist for a given view.
```

Comment:
What's the use of this? Can someone change property of the view ? if yes, then can that be part of the example ?

Reply:
Cannot change other properties of view, user will be allowed to change only the allowlist for the view.